HW7/arudova/task3.py

```diff
@@ -0,0 +1,8 @@
+def wordcount (word):
+    ''' this func returns count of words'''
+    output = {}
+    for k in word:
+        output.update({str(k): word.count(k)})
+    return output
+value = input('world = ')
+print (wordcount(value))
```

Comment:
another way:

def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result